static/app/utils/replays/getCurrentUserAction.tsx

```diff
@@ -4,9 +4,9 @@ import {Crumb} from 'sentry/types/breadcrumbs';
 export function getCurrentUserAction(
   userActionCrumbs: Crumb[] | undefined,
   startTimestamp: number | undefined,
-  currentHoverTime: number
+  currentHoverTime: number | undefined
 ) {
-  if (!startTimestamp || !userActionCrumbs) {
+  if (!startTimestamp || !userActionCrumbs || !currentHoverTime) {
```

Comment:
should be:
```suggestion
  if (!startTimestamp || !userActionCrumbs || currentHoverTime !== undefined) {
```
`0` is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), so the check `!currentHoverTime` is true when `currentHoverTime=0`. But we could hover right at the start and have a breadcrumb there too.